otcextensions/osclient/cbr/v3/backup.py

```diff
@@ -0,0 +1,428 @@
+        )
+        parser.add_argument(
+            '--sort',
+            metavar='sort',
+            help=_('A group of properties separated by commas (,)'
+                   ' and sorting directions.')
+        )
+        parser.add_argument(
+            '--start-time',
+            metavar='<start_time>',
+            help=_('Time when the backup starts.')
+        )
+        parser.add_argument(
+            '--status',
+            metavar='status',
+            choices=['available', 'protecting', 'deleting', 'restoring',
+                     'error', 'waiting_protect', 'waiting_delete',
+                     'waiting_restore'],
+            help=_('Status')
+        )
+        parser.add_argument(
+            '--used-percent',
+            metavar='<used_percent>',
+            help=_('Backups are filtered based on the '
+                   'occupied vault capacity.')
+        )
+        parser.add_argument(
+            '--vault-id',
+            metavar='<vault_id>',
+            help=_('Vault ID.')
+        )
+        return parser
+
+    def take_action(self, parsed_args):
+        client = self.app.client_manager.cbr
+
+        query = {}
+
+        if parsed_args.checkpoint_id:
+            query['checkpoint_id'] = parsed_args.checkpoint_id
+        if parsed_args.dec:
+            query['dec'] = parsed_args.dec
+        if parsed_args.end_time:
+            query['end_time'] = parsed_args.end_time
+        if parsed_args.image_type:
+            query['image_type'] = parsed_args.image_type
+        if parsed_args.limit:
+            query['limit'] = parsed_args.limit
+        if parsed_args.marker:
+            query['marker'] = parsed_args.marker
+        if parsed_args.member_status:
+            query['member_status'] = parsed_args.member_status
+        if parsed_args.name:
+            query['name'] = parsed_args.name
+        if parsed_args.offset:
+            query['offset'] = parsed_args.offset
+        if parsed_args.own_type:
+            query['own_type'] = parsed_args.own_type
+        if parsed_args.parent_id:
+            query['parent_id'] = parsed_args.parent_id
+        if parsed_args.resource_az:
+            query['resource_az'] = parsed_args.resource_az
+        if parsed_args.resource_id:
+            query['resource_id'] = parsed_args.resource_id
+        if parsed_args.resource_name:
+            query['resource_name'] = parsed_args.resource_name
+        if parsed_args.resource_type:
+            query['resource_type'] = parsed_args.resource_type
+        if parsed_args.sort:
+            query['sort'] = parsed_args.sort
+        if parsed_args.start_time:
+            query['start_time'] = parsed_args.start_time
+        if parsed_args.status:
+            query['status'] = parsed_args.status
+        if parsed_args.used_percent:
+            query['used_percent'] = parsed_args.used_percent
+        if parsed_args.vault_id:
+            query['vault_id'] = parsed_args.vault_id
```

Comment:
Maybe it will be better to put all parsed_args into list ([]) and iterate it? For example,
`        for arg in args_list:
            val = getattr(parsed_args, arg)
            if val:
                attrs[arg] = val`